pkg/stconfig/keygen.go

```diff
@@ -0,0 +1,19 @@
+package stconfig
```

Comment:
Can you add a license to this file?
A godoc for the package would also be welcome.